static/components/receipt.js

```diff
@@ -2,18 +2,28 @@ window.app.component('receipt', {
   name: 'receipt',
   props: ['data'],
   data() {
-    return {}
+    return {
+      currency: null,
+      exchangeRate: null
+    }
   },
   computed: {
     cartSubtotal() {
       let subtotal = 0
-      this.data.extra.details.items.forEach(item => {
-        subtotal += item.price * item.quantity
-      })
+      if (!this.data.extra.details || !this.data.extra.details.items) {
```

Comment:
nitpick: should use optional chaining (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)